src/integrationTest/java/uk/gov/hmcts/darts/notification/service/NotificationServiceTest.java

```diff
@@ -0,0 +1,214 @@
+package uk.gov.hmcts.darts.notification.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.darts.notification.dto.GovNotifyRequest;
+import uk.gov.hmcts.darts.notification.dto.SaveNotificationToDbRequest;
+import uk.gov.hmcts.darts.notification.entity.Notification;
+import uk.gov.hmcts.darts.notification.exception.TemplateNotFoundException;
+import uk.gov.hmcts.darts.notification.helper.TemplateIdHelper;
+import uk.gov.hmcts.darts.notification.repository.NotificationRepository;
+import uk.gov.service.notify.NotificationClientException;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@SpringBootTest
+@ActiveProfiles({"dev", "test"})
+@ExtendWith(MockitoExtension.class)
+class NotificationServiceTest {
+
+    public static final String TEST_EMAIL_ADDRESS = "test@test.com";
+    public static final String REQUEST_TO_TRANSCRIBER_TEMPLATE_NAME = "request_to_transcriber";
+    @Autowired
+    NotificationService service;
+
+    @Autowired
+    NotificationRepository notificationRepo;
+
+    @MockBean
+    TemplateIdHelper templateIdHelper;
+
+    @MockBean
+    GovNotifyService govNotifyService;
+
+    @BeforeEach
+    void beforeEach() {
+        notificationRepo.deleteAll();
+    }
+
+    @Test
+    void scheduleNotification() {
```

Comment:
Minor point, but could we give the test name abit more detail? Would be good to give it a clear name for what it is you are trying to achieve and what the result should be.